annotator/src/main/java/org/cbioportal/annotator/internal/GenomeNexusImpl.java

```diff
@@ -202,6 +203,7 @@ private AnnotationControllerApi initApiClient()
         if (this.genomeNexusBaseUrl != null && this.genomeNexusBaseUrl.length() > 0)
         {
             ApiClient client = new ApiClient();
+            client.setReadTimeout(300);
```

Comment:
aren't we supposed to use `READ_TIMEOUT_OVERRIDE ` here? Also should it be `300` or `300000`?

Reply:
@onursumer lol thanks, updating now